### ISSUE TITLE
ARIA hidden treated differently when capitalized

### ISSUE THREAD

**Issue by daniel-montalvo** [auto-wcag/auto-wcag#2272]
## ACT Rules

- [Element with aria-hidden has no content in sequential focus navigation](https://www.w3.org/WAI/standards-guidelines/act/rules/6cfa84/proposed/#accessibility-support)
- [Zoomed text node is not clipped with CSS overflow](https://www.w3.org/WAI/standards-guidelines/act/rules/59br37/proposed/#accessibility-support)

### Test case 1 ...

ARIA hidden capitalized

```html
<p>This is to test if ARIA hidden is treated differently if it's capitalized</p>

<div aria-hidden="true">This is aria-hidden true, no caps</div>
<div aria-hidden="TRUE">This is aria-hidden true, caps</div>
```

[OpenARIA hidden capitalized test case](https://codepen.io/daniel-montalvo/pen/QwLXYxe))

#### Test instructions

With a screen reader enabled, Use up and down arrow keys or left an right flicks.

Expected results are that none of the content with `aria-hidden="true"` or `aria-hidden="TRUE"` is read by screen readers.

Tested with:

- NVDA 2024.4
- JAWS 2025
- Chrome Version 132.0.6834.160 (Official Build) (64-bit)
- Edge Version 132.0.2957.140 (Official build) (64-bit)

## Test results

Please report test results in a comment below. Please use the following format:

```md
## Test case X

- Date: ...
- Operating system: ..., version ...
- Browser: ..., version ...
- Assistive technology: ..., version ...
- Test result: Passed / Failed (if failed, how was it different from expected?)
```

**Comment by daniel-montalvo** [comment]
We should probably remove this assertion from the rule as this no longer seems to happen.

**Comment by shunguoy** [comment]
Per [ARIA in HTML](https://w3c.github.io/html-aria/#case-sensitivity):
  

> Authors SHOULD use [ASCII lowercase](https://infra.spec.whatwg.org/#ascii-lowercase) for all role token values and any state or property attributes (aria-*) whose values are [defined as tokens](https://www.w3.org/TR/wai-aria-1.2/#propcharacteristic_value).
